recipes/tinyalign/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set version = "0.2" %}
+
+package:
+  name: tinyalign
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/t/tinyalign/tinyalign-{{ version }}.tar.gz
+  sha256: 33601ea7b3171d7fb495f59ef533d29fba7e0091dd69c1225ff1c2eed10b8a59
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - cython
+    - setuptools_scm
+  run:
+    - python
+
+test:
+  imports:
+    - tinyalign
+
+about:
+  home: https://github.com/marcelm/tinyalign/
+  license: MIT License
+  license_file: LICENSE
+  summary: 'Banded edit distance'
```

Taken (and shortened) from https://github.com/marcelm/tinyalign/blame/v0.2/README.md#L5-L7:
```suggestion
  summary: A small Python module providing edit distance and Hamming distance computation.
```
If you want to, you can add a detailed description too.